chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SearchViewController.java

```diff
@@ -393,4 +400,20 @@ public void download(@RequestParam(value="edition", required=false) String editi
 		return searchMenuManager.getPopulateSearchOptions(simple);
 	}
 	
+	@ApiOperation(value="Get all developer decertifications in the CHPL", 

```

Comment:
I could see it maybe going into the DeveloperController as /developers/decertified or something like that. That said, if we ever ended up with thousands of decertified developers we might decide to make it more of a "search" and do the filtering on the server side but we could always move the method again if that happened.